javascript/code-challenges/challenges-02.test.js

```diff
@@ -0,0 +1,317 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+
+CHALLENGE 1 - Review
+
+Write a function named raisedToTheThird that takes in an array of numbers and returns a new array of each of those numbers raised to the 3rd power (hint: look up Math.pow()). Use forEach to solve this problem.
+
+------------------------------------------------------------------------------------------------ */
+
+const raisedToTheThird = (arr) => {
+  const returnArray = [];
+  arr.forEach(num => {
+    returnArray.push(Math.pow(num, 3));
+
+  });
+  return returnArray;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named addOne that, given an array of numbers, uses map to return a new array with each value simply incremented by 1. 
+------------------------------------------------------------------------------------------------ */
+
+const addOne = (arr) => {
+  const incArr = arr.map(x => x + 1);
+  return incArr;
+};
```

Comment:
You could also just return arr.map, since it creates a new array no matter what.
```js
return arr.map(x => x + 1);
```